feedback/lib/src/feedback_widget.dart

```diff
@@ -294,3 +306,79 @@ class FeedbackWidgetState extends State<FeedbackWidget>
     FocusScope.of(context).requestFocus(FocusNode());
   }
 }
+
+const _screenshotId = 'screenshot_id';
+const _controlsColumnId = 'controls_column_id';
+const _sheetId = 'sheet_id';
+
+class _FeedbackLayoutDelegate extends MultiChildLayoutDelegate {
+  _FeedbackLayoutDelegate({
+    required this.query,
+    required this.sheetFraction,
+    required this.progress,
+  });
+
+  MediaQueryData query;
+  double sheetFraction;
```

Comment:
This is how much of the vertical screen the bottom sheet covers, right?

Reply:
Yep.